src/main/java/network/brightspots/rcv/DominionCvrReader.java

```diff
@@ -230,30 +232,48 @@ private void validateNamesAreInContest(List<CastVoteRecord> castVoteRecords)
   private void gatherCvrsForContest(List<CastVoteRecord> castVoteRecords, String contestIdToLoad) {
     try {
       Path singleCvrPath = Paths.get(cvrPath, CVR_EXPORT);
-      Path firstCvrPath = Paths.get(cvrPath, String.format(CVR_EXPORT_PATTERN, 0));
+
       if (singleCvrPath.toFile().exists()) {
         HashMap json = JsonParser.readFromFile(singleCvrPath.toString(), HashMap.class);
         parseCvrFile(json, castVoteRecords, contestIdToLoad);
-      } else if (firstCvrPath.toFile().exists()) {
+      } else {
+        // We are expecting multiple CvrExport_N.json files
+        Pattern pattern = Pattern.compile("^CvrExport_\\d+\\.json$");
+        HashSet<File> matchedCvrFiles = new HashSet<>();
+
+        // Use regex to find all CvrExport_N.json files
+        File[] files = Paths.get(cvrPath).toFile().listFiles();
+        if (files != null) {
+          for (File file : files) {
+            Matcher matcher = pattern.matcher(file.getName());
+            if (matcher.matches()) {
+              matchedCvrFiles.add(file);
+            }
+          }
+        }
```

Comment:
Everything above, up to the else, can be simplified with:

```
    String regexPath = CVR_EXPORT_PATTERN.replaceAll("%d", "\\\\d+");
    File cvrDirectory = new File(cvrPath);
    File[] files = cvrDirectory.listFiles((dir, name) -> name.matches(pattern));
```

which has the added benefit of not duplicating the `Cvr_Export` string